src/AppInstallerCLICore/Commands/ExportCommand.cpp

```diff
@@ -53,11 +53,14 @@ namespace AppInstaller::CLI
 
     void ExportCommand::ExecuteInternal(Execution::Context& context) const
     {
+        context.SetFlags(Execution::ContextFlag::TreatSourceFailuresAsWarning);
+
         context <<
             Workflow::ReportExecutionStage(Workflow::ExecutionStage::Discovery) <<
             Workflow::OpenSource <<
             Workflow::OpenCompositeSource(Repository::PredefinedSource::Installed) <<
             Workflow::SearchSourceForMany <<
+            Workflow::HandleSearchResultFailures <<
```

Comment:
>HandleSearchResultFailures [](http://example.com/codeflow?start=22&length=26)

This is called after every SearchSourceForMany or SearchSourceForSingle, shall we consolidate them inside?

Reply:
I had considered that, but I'm not sure.  I think that there is probably a case to be made to have an uber search task (or two) that takes a few well defined behavior flags and roll search + handle failure + ensure* since they are very much linked.

But given our timeframe I'm going to put that on the debt rather than change it here.